playbooks/provisioning/openstack/README.md

```diff
@@ -19,6 +19,16 @@ There are no additional dependencies for the cluster nodes. Required
 configuration steps are done by Heat given a specific user data config
 that normally should not be changed.
 
+## Required galaxy modules
+
+In order to pull in external dependencies for DNS configuration steps,
+the following commads need to be executed:
+
+    #FIXME(bogdando) use the requirements file when it's ready at redhat-cop/infra-ansible
+    #ansible-galaxy install -r galaxy-requirements.yml -p <...>
```

Comment:
Can you uncomment this and make it the default way to set up the roles? With the `galaxy-requirements.yaml` changes, that should work.